homepageapp/templates/registration/index.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'homepageapp/base.html' %}
+
+{% block content %}
+
+    {% for message in messages %}
+    {% csrf_token %}
+        <strong>Message: </strong> {{ message }}
```

Comment:
```suggestion
        <p><strong>Message: </strong> {{ message }}</p>
```
there can be multiple messages, this will put each one in separate line.